Towny/src/main/resources/plugin.yml

```diff
@@ -10,7 +10,7 @@ description: >
     Resident-Town-Nation hierarchy combined with a grid based
     protection system, many features and an expansive API.
 
-softdepend: [MultiVerse,Multiverse-Core,XcraftGate,My_Worlds,Citizens,PersonalWorlds,WormholeXTremeWorlds,Vault,Reserve,MultiWorld,TheNewChat,PlaceholderAPI,Essentials,Hyperverse,GroupManager,iConomy,LuckPerms]
+softdepend: [MultiVerse,Multiverse-Core,XcraftGate,My_Worlds,Citizens,PersonalWorlds,WormholeXTremeWorlds,Vault,Reserve,MultiWorld,TheNewChat,PlaceholderAPI,Essentials,Hyperverse,GroupManager,iConomy,LuckPerms,CoreProtect]
```

Comment:
Do we actually need CoreProtect to be in the softdepend? I'm just thinking if we can not having it we might avoid the pesky circular loading detection.

Reply:
Yes, since we're using coreprotect's api/classes, circular loading only becomes a concern when they add us to their plugin.yml.

Reply:
Yeah I was more concerned about a 3rd party plugin that has both Towny and CoreProtect in their plugin.yml, causing a loop because of 3-4 plugins being present. I suppose if it ever becomes an issue we can stick it all in its own class that only gets initialized after we know coreprotect is there.